examples/js/nodes/inputs/FloatNode.js

```diff
@@ -29,6 +29,14 @@ Object.defineProperties( THREE.FloatNode.prototype, {
 	}
 } );
 
+THREE.FloatNode.prototype.generateReadonly = function ( builder, output, uuid, type, ns, needsUpdate ) {
+
+	var value = this.number;
+
+	return builder.format( ~~value !== value ? value : value + ".0", type, output );
```

Comment:
Minor thing: I don't think using these bitwise operations are worth it these days (speed vs readability).

Reply:
Instead what do you recommend `parseInt` or `Math.floor`?

Reply:
I think that `Math.floor` is "more" number-oriented.